README.md

```diff
@@ -53,7 +58,7 @@ var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
 How would I go about accessing the string "pizza" in the above array?
 
 ```js
-// write code here
+create a for loop for foods.. now you can use an array for pizza [4]; which is the 4th in the array
```

Comment:
Here, all we are trying to do is access a value from the nested data structure `foods`. We do not need to loop, in this case we know that `foods` is actually an array of three arrays, and the value `"pizza"` is the first item, in the second array in `foods`. So to grab just that value `var pizza = foods[1][0]`